zeppelin-zengine/src/test/java/org/apache/zeppelin/interpreter/SparkInterpreterModeTest.java

```diff
@@ -137,6 +149,10 @@ public void testYarnClusterMode() throws IOException, YarnException, Interrupted
     interpreterSettingManager.close();
   }
 
+  private boolean isSpark2(String sparkVersion) {
+    return sparkVersion.startsWith("2.");
```

Comment:
use SparkVersion.java?

Reply:
SparkVersion.java is in spark module, can not be used in zeppelin-zengine, I don't want to make zeppelin-zengine depends on spark module just for this kind of trivial thing.